packages/edit-site/src/store/test/actions.js

```diff
@@ -76,29 +75,6 @@ describe( 'actions', () => {
 		} );
 	} );
 
-	describe( 'removeTemplate', () => {
```

Comment:
I removed the unit test because I don't think it's serving any purpose besides slowing us down. We could add it back if the action becomes more complicated. Or if anyone thinks otherwise.

Reply:
Yeah, agree with removing it. There are already tests for deleteEntityRecord, and the notices should ideally be tested with e2e tests if they're to be tested.